src/SFA.DAS.ApprenticeCommitments.Api/SFA.DAS.ApprenticeCommitments.Api.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <Content Remove="nlog.Debug.config" />
+      <Content Remove="nlog.Release.config" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <None Include="nlog.Debug.config">
+        <IsTransformFile>true</IsTransformFile>
+        <DependentUpon>nlog.config</DependentUpon>
+      </None>
+      <None Include="nlog.Release.config">
+        <IsTransformFile>true</IsTransformFile>
+        <DependentUpon>nlog.config</DependentUpon>
+      </None>
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="MediatR" Version="6.0.0" />
+        <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="6.0.1" />
+        <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.8" />
+        <PackageReference Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Version="2.2.0" />
+        <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="2.2.0" />
+        <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions" Version="2.2.0" />
+        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.7.1" />
+        <PackageReference Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="2.2.0" />
+        <PackageReference Include="Microsoft.Extensions.Options" Version="3.1.5" />
+        <PackageReference Include="NLog.Web.AspNetCore" Version="4.8.0" />
+        <PackageReference Include="SFA.DAS.Api.Common" Version="1.0.8" />
+        <PackageReference Include="SFA.DAS.NLog.Targets.Redis" Version="1.1.5" />
+        <PackageReference Include="SFA.DAS.Configuration.AzureTableStorage" Version="3.0.84" />
+        <PackageReference Include="Swashbuckle.AspNetCore" Version="5.2.1" />
+    </ItemGroup>
+    <ItemGroup>
+        <Content Update="appsettings.json">
+            <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+        </Content>
+        <Content Update="appsettings.Development.json">
+            <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+        </Content>
+        <Content Update="nlog.config">
+            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+        </Content>
+        <Content Update="nlog.config">
+          <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+          <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+          <TransformOnBuild>true</TransformOnBuild>
+        </Content>
+    </ItemGroup>
```

Comment:
curious why we have 2 content sections for nlog.config. I don't really recognise the second section (well I've not seen that type of block before)

Reply:
That's how it was in the `EmployerIncentives` project that was used as inspiration.  Looking at it, seems to be unnecessary - the second section will overwrite the first section.